src/app/client/src/app/modules/groups/components/activity/activity-details/activity-details.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="sbt-center-container sbt-back-header sbt-mygroups-details relative9">
+<div class="mat-inside-page-container mat-app-background2 sbt-back-header sbt-mygroups-details relative9">
```

Comment:
is this class requied " sbt-back-header "

Reply:
removed